etc/stack-schema-map.yaml

```diff
@@ -0,0 +1,7 @@
+# alignment of stack with beats and ecs versions
+#    ECS versions do not align perfectly with stack releases (as of 7.13), so this will reflect MAX ecs version for a
+#    given release
+
+"7.13.0":
+  beats: "7.12.0"
```

Comment:
this right?

Reply:
beats release about the same time as the stack, so we cannot update this until it is released. In the mean time it will defer to the latest non-master we have saved (7.12.0 and 1.9.0 in this case). I'll comment this out for now

Reply:
4c54015